src/render/md.jl

```diff
@@ -77,51 +69,62 @@ function writemime(io::IO, mime::MIME"text/md", ents::Entries, config::Config)
     if !isempty(exported.entries)
         print_help(io, config.mdstyle_exported, "Exported")
         for (modname, obj, ent) in exported.entries
-            writemime(io, mime, modname, obj, ent, config)
+            writemd(io, modname, obj, ent, config)
         end
     end
     if !isempty(internal.entries)
         print_help(io, config.mdstyle_internal, "Internal")
         for (modname, obj, ent) in internal.entries
-            writemime(io, mime, modname, obj, ent, config)
+            writemd(io, modname, obj, ent, config)
         end
     end
 end
 
-function writemime{category}(io::IO, mime::MIME"text/md", modname, obj, ent::Entry{category},
-                                                                              config::Config)
+function writemd{category}(io::IO, modname, obj, ent::Entry{category}, config::Config)
     objname = writeobj(obj, ent)
     println(io, "---\n")
     print_help(io, config.mdstyle_objname, objname)
-    writemime(io, mime, docs(ent))
+    writemd(io, docs(ent))
     println(io)
     for k in sort(collect(keys(ent.data)))
         print_help(io, config.mdstyle_meta, "$k:")
-        writemime(io, mime, Meta{k}(ent.data[k]))
+        writemd(io, Meta{k}(ent.data[k]))
         println(io)
     end
 end
 
-function writemime(io::IO, mime::MIME"text/md", md::Meta)
+function writemd(io::IO, md::Meta)
     println(io, md.content)
 end
 
-function writemime(io::IO, mime::MIME"text/md", m::Meta{:parameters})
-    for (k, v) in m.content
-        println(io, k)
-    end
-    writemime(io, mime, v)
-end
-
-function writemime(io::IO, ::MIME"text/md", m::Meta{:source})
+function writemd(io::IO, m::Meta{:source})
     path = last(split(m.content[2], r"v[\d\.]+(/|\\)"))
     println(io, "[$(path):$(m.content[1])]($(url(m)))")
 end
 
-function header(io::IO, ::MIME"text/md", doc::Metadata, config::Config)
+function headermd(io::IO, doc::Metadata, config::Config)
     print_help(io, config.mdstyle_header, doc.modname)
 end
 
-function footer(io::IO, ::MIME"text/md")
+function footermd(io::IO)
     println(io, "")
 end
+
+
+#  =============== TODO: if no Problems arise delete this below
+#length(ents::Entries) = length(ents.entries)  # TODO: if no Problems arise delete this
+
+
+#function writemd(io::IO, mime::MIME"text/md", manual::Manual)
+#    println(""">>SEEMS IT GETS NOT CALLED: writemd(io::IO, mime::MIME"text/md", manual::Manual) is this done in line 36??""")
+#    for page in pages(manual)
+#        writemd(io, MIME("text/md"), docs(page))
+#    end
+#end
+#function writemd(io::IO, mime::MIME"text/md", m::Meta{:parameters})
+#    for (k, v) in m.content
+#        println(io, k)
+#    end
+#    println("""=====2 """, @which writemd(io, MIME("text/md"), v))
+#    writemd(io, MIME("text/md"), v)
+#end
```

Comment:
ok - maybe this should get at sometimes in the test case - as neighter build.jl for Lexicon/Docile nore there tests seems to call it.